boost-maven/boosters/booster-jaxrs-ee7/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The plugin pom.xml(s) at least should have copyrights.     Perhaps debatable if the IT poms are as big a deal (though easy enough to add them to all).

Reply:
I just checked ci.maven and we do not have a copyright in the pom.xml. So maybe it needs to be added in both places?

Reply:
I'm not sure of a hard and fast rule.. it seems like a good place to draw the line to me, though.